### ISSUE TITLE
Training of TARS model based on xlm-roberta-base not supported?

### ISSUE THREAD

**Issue by ann-so** [flairNLP/flair#3049]
In the [tutorial](https://github.com/flairNLP/flair/blob/master/resources/docs/TUTORIAL_10_TRAINING_ZERO_SHOT_MODEL.md) to train a TARS model, I replaced the bert-base-uncased embeddings with xml-roberta-base embeddings. 

```
# 5a: alternatively, comment out previous line and comment in next line to train a new TARS model from scratch instead
# tars = TARSClassifier(embeddings="bert-base-uncased")
tars = TARSClassifier(embeddings="xlm-roberta-base")
```

No error or warning message shows up during training, but loss is not really decreasing, and predictions seem to be empty, which results in 0.0 evaluation scores. 

see loss:
```
EPOCH	TIMESTAMP	BAD_EPOCHS	LEARNING_RATE	TRAIN_LOSS	DEV_LOSS	DEV_PRECISION	DEV_RECALL	DEV_F1	DEV_ACCURACY
1	13:34:20	0	0.0200	0.6781343320539873	0.5536978244781494	0.0	0.0	0.0	0.0
2	13:36:27	0	0.0200	0.6408121450934816	0.5375534296035767	0.0	0.0	0.0	0.0
3	13:38:33	0	0.0200	0.6377008462209723	0.5173110365867615	0.0	0.0	0.0	0.0
4	13:40:40	1	0.0200	0.6371011527674117	0.5206502676010132	0.0	0.0	0.0	0.0
5	13:42:46	2	0.0200	0.6370353295454698	0.5219637155532837	0.0	0.0	0.0	0.0


```
see predictions:
```
How far is it from Denver to Aspen ?
 - Gold: question about number
 - Pred: 
 -> MISMATCH!

What county is Modesto , California in ?
 - Gold: question about location
 - Pred: 
 -> MISMATCH!
```
Are other TransformerEmbeddings not supported for TARS models?

**Comment by alanakbik** [comment]
Hello @ann-so, yes it also works with other transformers. However, the process of fine-tuning transformers is typically very sensitive to the chosen learning rate. Often, with the wrong learning rate, the models learn nothing. You could try increasing the learning rate, for instance by an order of magnitude. 

Instead of using the` .train` method that uses SGD and a relatively large learning rate, you could also use the `fine_tune` method (see info [here](https://github.com/flairNLP/flair/blob/master/resources/docs/TUTORIAL_7_TRAINING_A_MODEL.md#training-a-named-entity-recognition-ner-model-with-transformers)) which uses AdamW and a much smaller learning rate. I.e. use this snippet and try learning rates 5.0e-5 and 5.0e-6.

```python
# 8. start the training
trainer.train(base_path='resources/taggers/trec',  # path to store the model artifacts
                  learning_rate=5.0e-6,
                  mini_batch_size=4,
                  mini_batch_chunk_size=1,  # remove this parameter to speed up computation if you have a big GPU
                  )
```

**Comment by ann-so** [comment]
Thanks @alanakbik , it works now fine with the fine_tune-Funktion.
